server/src/main/java/org/apache/druid/metadata/SQLMetadataSegmentManager.java

```diff
@@ -757,21 +715,23 @@ public DataSegment map(int index, ResultSet r, StatementContext ctx) throws SQLE
               .addSegmentIfAbsent(segment);
         });
 
-
-    replaceDataSourcesSnapshot(newDataSources.entrySet()
-                                             .stream()
-                                             .collect(Collectors.toMap(
-                                                 e -> e.getKey(),
-                                                 e -> e.getValue().toImmutableDruidDataSource()
-                                             )));
-  }
-
-  /**
-   * replace "dataSourcesSnapshot" atomically
-   */
-  private void replaceDataSourcesSnapshot(Map<String, ImmutableDruidDataSource> dataSources)
-  {
-    dataSourcesSnapshot = new DataSourcesSnapshot(dataSources);
+    /**
```

Comment:
Javadoc comment can't be inside a method

Reply:
changed to regular comment